creator-node/src/monitors/MonitoringQueue.js

```diff
@@ -0,0 +1,111 @@
+const Bull = require('bull')
+const redis = require('../redis')
+const config = require('../config')
+const { logger } = require('../logging')
+const { getDatabaseSize } = require('./database')
+
+const MONITORING_REDIS_PREFIX = 'monitoring'
+const PROCESS_NAMES = Object.freeze({
+  monitor: 'monitor'
+})
+
+/**
+ * List of all monitors to run, containing:
+ *  @param {string} name A unique name (for caching in redis)
+ *  @param {number} ttl TTL in seconds for how long a cached value is good for
+ *  @param {function} func The actual work to compute a value. The return value is what is cached.
+ */
+const MONITORS = [
+  {
+    name: 'databaseSize',
+    ttl: 120,
+    func: getDatabaseSize
```

Comment:
i like this pattern a lot - can you share that extended MONITORS code you showed us?

Reply:
Sweet! Yeah -- here's the PR. wanted to get this reviewed and good to go first 
https://github.com/AudiusProject/audius-protocol/pull/1195